src/azure-cli/azure/cli/command_modules/databoxedge/generated/_params.py

```diff
@@ -39,19 +39,22 @@ def load_arguments(self, _):
         c.argument('location', arg_type=get_location_type(self.cli_ctx), required=False,
                    validator=get_default_location_from_resource_group)
         c.argument('tags', tags_type)
-        c.argument('sku', action=AddSku, nargs='+', help='The SKU type.')
-        c.argument('etag', type=str, help='The etag for the devices.')
-        c.argument('data_box_edge_device_status', options_list=['--status'], arg_type=get_enum_type(['ReadyToSetup',
-                                                                                                     'Online',
-                                                                                                     'Offline',
-                                                                                                     'NeedsAttention',
-                                                                                                     'Disconnected',
-                                                                                                     'PartiallyDisconnected',
-                                                                                                     'Maintenance']),
-                   help='The status of the Data Box Edge/Gateway device.')
-        c.argument('description', type=str, help='The Description of the Data Box Edge/Gateway device.')
-        c.argument('model_description', type=str, help='The description of the Data Box Edge/Gateway device model.')
-        c.argument('friendly_name', type=str, help='The Data Box Edge/Gateway device name.')
+        c.argument('sku', options_list=['--sku'], help='The SKU type.')
```

Comment:
unnecessary to add options_list.
Another question: why do we remove `nargs='+'`?